inject-java/build.gradle

```diff
@@ -10,6 +10,7 @@ dependencies {
 
     testImplementation project(':aop')
     testAnnotationProcessor project(":inject-java")
+    testImplementation project(":inject-java-test")
```

Comment:
inject-java-test has an api dependency on inject-java, thus creating a cycle?

Reply:
Seems to work fine 🤷‍♂️